src/main.cpp

```diff
@@ -1223,6 +1236,7 @@ int main(int argc, char**argv)
   {
     realm_manager->stop();
     delete realm_manager; realm_manager = NULL;
+    delete hss_peer_connection_alarm; hss_peer_connection_alarm = NULL;
```

Comment:
I think you delete this alarm in the realm manager - you shouldn't delete it again here.

Reply:
I think I'll delete it here instead to be consistent with the other objects such as the Diameter resolver.